dockerfiles/e2e-saas/docker-entrypoint.sh

```diff
@@ -165,5 +174,17 @@ fi
 echo "Installing upstream dependency."
 npm --silent i ../../e2e/
 
+# create a folder and record a video
+ffmpeg_path="./report/ffmpeg_report"
```

Comment:
would be nice to extract this into the meaningful functions e.g. strartRecording / stopRecording

Reply:
Done.